package.json

```diff
@@ -21,7 +21,8 @@
     "semantic-release": "semantic-release pre && npm publish && semantic-release post",
     "snapshot": "jest --updateSnapshot",
     "start": "start-storybook -p 6006",
-    "test": "jest --coverage"
+    "test": "jest --coverage",
+    "test-debug": "node --inspect-brk node_modules/.bin/jest --runInBand --coverage"
```

Comment:
Super nitty and I don't feel that strongly either way (nor do I think it matters that much): `test-debug` feels a little backwards - I get that it's trying to keep with the `test` paradigm, but `debug-test` feels a lot more natural. Just my ✌💰.

**tl;dr: it doesn't really matter**

Reply:
Hah, I actually had it like that at first, but changed it to test-debug because that's how it was in [insights](https://github.com/edx/edx-analytics-dashboard/blob/master/gulpfile.js#L40). Doesn't really matter though, I can make it debug-test.